### PR TITLE
Refactor panel page with support for hybrids v7

### PR DESCRIPTION
**Pull request by smalluban**
The main goal is to support the current major version of the hybrids (v7). Additionally, using this opportunity I've updated the page structure and component definitions.

The UI changes were tested on desktop Chrome, macOS Safari, and iOS simulator - iPhone 13 mini, iPhone 13, iPad 11 (3rd gen). Every device/browser shows the popup differently, so there are some clever hacks to support all of them with only one code base. The main problem is with window sizing, as on Chrome it tries to shrink it as possible, on macOS Safari it stays as it should. On another hand, on iOS, there are other size limitations than on desktop. iPhone has less than 400px width and shows popup in two modes - half open and fully open. We want to support correctly UI in both cases, which made styling even harder. However, I found a tricky hack with the media query, which covers the case ;)

I also checked the longest supported language (de) for the main title of the simple view, so there were required some CSS changes to the font sizing of the header to fit in on the screen (especially on iPhone mini).

The PR, for now, relates to the GitHub branch of the `webextension-packages`, we have to merge the corresponding PR there first and release a new package to the npm.

### CODE REVIEW

File: src/pages/panel/components/layout.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  `.css`
+    :host {
+      display: flex;
+      flex-flow: column nowrap;
+      min-height: 100%;
+      width: 100%;
+      min-width: 375px;
+    }
+
+    @media only screen and (min-height: 346px) {
```
Comment (smalluban) [inline src/pages/panel/components/layout.js:40]: Here is the hack - on chrome we cannot just pass `height: 100%` as it shrinks the view to less then 100px height, but as the height is related to the content, it only applies when Chrome already calculates the height, so in the result using media does not break it. And what most important - this rule is required on macOS Safari to limit content height and display scrollbar correctly.

**Comment by smalluban** [comment]
@chrmod I think it would be great if you could run it locally, to see the changes.

**Comment by smalluban** [comment]
@chrmod @philipp-classen Roboto font is back, so I think the PR is ready for review and merge :)

**Review by philipp-classen — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -29,8 +29,8 @@
   "dependencies": {
     "@cliqz/adblocker": "1.22.7",
     "@cliqz/adblocker-webextension-cosmetics": "^1.22.7",
-    "@whotracksme/webextension-packages": "^0.1.1",
-    "hybrids": "^6.1.0",
+    "@whotracksme/webextension-packages": "github:whotracksme/webextension-packages#fix-hybrids-v7",
```
Comment (chrmod) [inline package.json:32]: `@whotracksme/webextension-packages@0.1.2` is released

File: src/pages/panel/components/layout.js (hunk 1 above)
Comment (chrmod) [reply src/pages/panel/components/layout.js:40]: sweet!

**Review by chrmod — COMMENTED** [review]
Look great!

**Review by chrmod — APPROVED** [review]

**Pull request merged**